### ISSUE TITLE
includeNotices ignored from config file

### ISSUE THREAD

**Issue by smustgrave** [pa11y/pa11y#638]
If includeNotices True is included in a json configuration file it's ignored.

**Comment by smustgrave** [comment]
Seems to not work if I have pa11y installed globally.  But locally (.node-modules/pa11y/bin/pa11y.js it works)

**Comment by josebolos** [comment]
Hi @smustgrave 

Could you please attach a config file so we can try to reproduce the issue and include in the issue description the result of running `pa11y --environment`?

**Comment by smustgrave** [comment]
{
  "defaults": {
    "standard": "WCAG2AA",
    "runners": ["axe"],
    "timeout": 90000,
    "level": "none",
    "wait": 2000,
    "chromeLaunchConfig": {
      "ignoreHTTPSErrors": false,
      "args": ["--no-sandbox"]
    },
    "ignore": [
      "Section508.D.H49.AlignAttr",
      "Section508.D.HeadingOrder",
      "WCAG2A.Principle1.Guideline1_3.1_3_1.H49.AlignAttr",
      "WCAG2A.Principle1.Guideline1_3.1_3_1.G141",
      "WCAG2AA.Principle1.Guideline1_3.1_3_1.H49.AlignAttr",
      "WCAG2AA.Principle1.Guideline1_3.1_3_1.G141",
      "color-contrast",
      "heading-order",
      "WCAG2AAA.Principle1.Guideline1_3.1_3_1.H49.AlignAttr",
      "WCAG2AAA.Principle1.Guideline1_3.1_3_1.G141"
    ],
    "includeWarnings": true,
    "includeNotices": true
  }
}

  System:
    OS: macOS 12.3
    CPU: (10) arm64 Apple M1 Max
    Memory: 3.69 GB / 64.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 17.6.0 - /opt/homebrew/bin/node
    Yarn: 1.22.17 - /usr/local/bin/yarn
    npm: 8.6.0 - /opt/homebrew/bin/npm
  pa11y: 6.2.1

**Comment by josebolos** [comment]
@smustgrave you seem to be using `pa11y` but the config format that you pasted is for `pa11y-ci`. Does pa11y work fine with the following config file?

```
{
	"standard": "WCAG2AA",
	"runners": ["axe"],
	"timeout": 90000,
	"level": "none",
	"wait": 2000,
	"chromeLaunchConfig": {
		"ignoreHTTPSErrors": false,
		"args": ["--no-sandbox"]
	},
	"ignore": [
		"Section508.D.H49.AlignAttr",
		"Section508.D.HeadingOrder",
		"WCAG2A.Principle1.Guideline1_3.1_3_1.H49.AlignAttr",
		"WCAG2A.Principle1.Guideline1_3.1_3_1.G141",
		"WCAG2AA.Principle1.Guideline1_3.1_3_1.H49.AlignAttr",
		"WCAG2AA.Principle1.Guideline1_3.1_3_1.G141",
		"color-contrast",
		"heading-order",
		"WCAG2AAA.Principle1.Guideline1_3.1_3_1.H49.AlignAttr",
		"WCAG2AAA.Principle1.Guideline1_3.1_3_1.G141"
	],
	"includeWarnings": true,
	"includeNotices": true
}
```

**Comment by smustgrave** [comment]
Still doesn't work.  But if I install a local pa11y and run ./node-modules/pa11y/bin/pa11y.js it works.

**Comment by josebolos** [comment]
The version that you're using is 6.2.1 but the last version published, which is the version in the repo, is 6.2.2. Have you tried to install the latest version and check if you're experiencing the same issue?

The config that I pasted below works fine here when testing https://pa11y.org using pa11y 6.2.2, so the alternative is that it's something specific to the site that you are testing.
